Gemfile

```diff
@@ -84,7 +84,7 @@ group :development, :test do
   ##
   # We want to use the same version of rubocop as Codeclimate does - see
   # .codeclimate.yml and https://docs.codeclimate.com/docs/rubocop
-  gem 'rubocop', '~> 0.87.1', require: false
+  gem 'rubocop', '~> 0.89.1', require: false
```

Comment:
also needs updating in `.codeclimate.yaml`

Can probably go to all the way to 0.90 now